src/features/nft/portfolio/BidsModal.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import Modal from '../nftModals/modal'
+import avatar1 from 'common/assets/images/mock/avatar-1.png'
+import avatar2 from 'common/assets/images/mock/avatar-2.png'
+import avatar3 from 'common/assets/images/mock/avatar-3.png'
+import dayjs from 'dayjs'
+
+export type TBidsModal = {
+  isOpen: boolean
+  handleClose: () => void
+}
+
+const BidsModal: React.FC<TBidsModal> = ({ isOpen, handleClose }) => {
+  const comments = [
+    {
+      id: 1,
+      author: {
+        avatar: avatar1,
+        name: 'Ben Mingo',
+        username: '@zndrson',
+      },
+      bidPrice: '12,000PSL',
+      publishedAt: new Date(),
+    },
+    {
+      id: 2,
+      author: {
+        avatar: avatar2,
+        name: 'The Noc Design',
+        username: '@zndrson',
+      },
+      bidPrice: '12,000PSL',
+      publishedAt: new Date(),
+    },
+    {
+      id: 3,
+      author: {
+        avatar: avatar3,
+        name: 'Michael Brewer',
+        username: '@zndrson',
+      },
+      bidPrice: '12,000PSL',
+      publishedAt: new Date(),
+    },
+  ]
+
+  const title = (
+    <div>
+      <div className='text-h2'>Bids activity (8 bids)</div>
+      <div className='text-h5 text-gray-71'>NFT ID 230456346</div>
+    </div>
+  )
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      handleClose={() => handleClose()}
+      size='564px'
+      title={title}
```

Comment:
NB: Could you inline title JSX content here instead of variable?

In this case looks fine, but IMO it's a bad practice and if we have few such variables the component become much harder to read, so you see `title` variable and then should find where is it used

Reply:
fixed